README.md

```diff
@@ -59,6 +59,30 @@ The gem offers the following rake tasks that can be manually run according to yo
 - `rails db:rollback_branches` - run it to manually rolls back phantom migrations.
 - `rails db:phantom_migrations` - displays a list of phantom migrations.
 
+### Local usage
+
+If you're enough of a geek to want to develop with your beloved gems alongside the general Gemfile but separate from it, you could leverage this trick:
```

Comment:
Can this be rephrased so that it doesn't include possibly offensive words, such as geek?

[For reference](https://www.cjr.org/language_corner/geek.php#:~:text=Keep%20in%20mind%20that%20%E2%80%9Cgeek%E2%80%9D%20can%20still%20be%20an%20insult%2C%20depending%20on%20the%20target%20and%20context.%20Using%20it%20to%20refer%20to%20oneself%20or%20to%20someone%20you%20know%20embraces%20the%20label%20is%20always%20the%20safest%20way%20to%20avoid%20unintended%20pain.)